package.json

```diff
@@ -4,6 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start": "NODE_ENV=development nodemon src/index.js",
+    "db-init": "sequelize init",
+    "dummy-model":"npx sequelize-cli model:generate --name User --attributes firstName:string,lastName:string,email:string",
```

Comment:
@hezronkimutai by default Postgres create table in lowercase `user` why you change it to plural case.

Reply:
Okey, Let me change that.